### PR TITLE
hit the API

### PR DESCRIPTION
**Pull request by Abel-desu**
- [ ] **In this project I make** 
- Send and receive data from an API
- working with JSON
- JavaScript code asynchronous

### CODE REVIEW

File: src/RecentScore.js [hunk 1]
```diff
@@ -31,4 +31,38 @@ const RecentScores = [
...
+{
+  "name": "nadia",
+   "score": 20
+},
+{
+  "name": "nadia",
+   "score": 20
+},
+{
+  "name": "nadia",
```
Comment (tufoinnkuo10) [inline src/RecentScore.js]: - [ ] scores should come from the API,,, kindly remove this

File: src/index.js [hunk 2]
```diff
@@ -1,10 +1,44 @@
 import RecentScores from './RecentScore';
 import './style.css';
-
+// const form = document.getElementById('form');
```
Comment (tufoinnkuo10) [inline src/index.js]: - [ ] kindly remove comments for cleaner code practice

File: src/score.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "name": "nadia",
+     "score": 20
+},
+{
+    "name": "nadia",
+     "score": 20
+},
+{
+    "name": "nadia",
+     "score": 20
```
Comment (tufoinnkuo10) [inline src/score.json:20]: - [ ] not necessary as everything comes from the api

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Abel-desu 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ board updates automatically after refresh
✅ API works well

## Required Changes ♻️♻️
Check the comments under the review


tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by topeogunleye — APPROVED** [review]
### Hi @Abel-desu, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- PR has a good title :heavy_check_mark:

> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**